pth_yadg.user.js

```diff
@@ -3,7 +3,7 @@
 // @name           RED YADG
 // @description    This script provides integration with online description generator YADG (http://yadg.cc) - Credit to Slack06
 // @license        https://github.com/SavageCore/yadg-pth-userscript/blob/master/LICENSE
-// @version        1.6.3
+// @version        1.6.4
```

Comment:
To remove

Reply:
Have removed the version bump, Please merge, Thanks a lot !